app/utils.py

```diff
@@ -25,12 +26,15 @@ def keys_to_snake_case(d):
     return new_dict
 
 
-def dict_to_model(movie_dict: dict):
+def dict_to_model(params: HashableParams, movie_dict: dict):
     """ converts a movie_dict to model """
+    # obtain datetime to use for saved_on
+    saved_on = datetime.today() - timedelta(days=int(params["page"]))
```

Comment:
This is overkill to me. Let's just save the page number alongside the movie. So we can just run `.order_by(date_created, page_num)`

Reply:
Not worth it... cuz then old movies that have not been retrieved for a while might have their original page numbers
Every movie was once on page 1... think about it

Reply:
That's where the second filter comes in to play by date

Reply:
What second filter?, this is the only filter

Reply:
something like `db.query(models.Movie).order_by(models.Movie.date_created.desc, models.Movie.page.desc)`. This would automatically get the latest movies and sort them by page

https://stackoverflow.com/questions/15791760/how-can-i-do-multiple-order-by-in-flask-sqlalchemy